packages/volto/src/components/manage/Blocks/Block/EditBlockWrapper.jsx

```diff
@@ -99,6 +107,13 @@ const EditBlockWrapper = (props) => {
           }}
           {...draginfo.dragHandleProps}
           className="drag handle wrapper"
+          aria-label={
+            intl.formatMessage(messages.drag) +
+            ' ' +
+            intl.formatMessage(messages.block) +
+            ' ' +
+            type
```

Comment:
i think it's better defining a tanslation with parameters, for example: 

`intl.formatMessage(messages.drag_block,{type}) `